tests/models/gemma/test_modeling_gemma.py

```diff
@@ -570,16 +571,24 @@ def test_model_2b_fp16_static_cache(self):
     @require_read_token
     def test_model_2b_bf16(self):
         model_id = "google/gemma-2b"
-        EXPECTED_TEXTS = {
-            7: [
-                "Hello I am doing a project on the 1990s and I need to know what the most popular music",
-                "Hi today I am going to share with you a very easy and simple recipe of <strong><em>Khichdi",
-            ],
-            8: [
-                "Hello I am doing a project on the 1990s and I need to know what the most popular music",
-                "Hi today I am going to share with you a very easy and simple recipe of <strong><em>Kaju Kat",
-            ],
-        }
+        if IS_ROCM_SYSTEM:
```

Comment:
Can you refactor this block so that one uses directly the dict without the if / else statement? 🙏

Reply:
Done!